pom.xml

```diff
@@ -78,6 +79,11 @@
             <version>${maven3.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>${apache-commons-version}</version>
```

Comment:
Are you adding a new dependency just for StringUtils.isNotBlank which is just `string != null && !string.trim().isEmpty()`?